src/main/java/serenitylabs/tutorials/vetclinic/playingball/model/SchoolSportsSchedule.java

```diff
@@ -0,0 +1,35 @@
+package serenitylabs.tutorials.vetclinic.playingball.model;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.Month;
+
+/**
+ * Created by kimahale on 9/8/2016.
+ */
+public class SchoolSportsSchedule implements GameSchedule {
+    @Override
+    public PlayBall forGameOn(LocalDate currentDay) {
+
+        //System.out.println(currentDay.getMonth().toString());
+
+        if (currentDay.getMonth() == Month.JUNE
+                        || currentDay.getMonth() == Month.JULY
+                        || currentDay.getMonth() == Month.AUGUST
+                        || currentDay.getMonth() == Month.SEPTEMBER) {
+            return new PlayCricket();
+        }
+
+        if (currentDay.getDayOfWeek() == DayOfWeek.WEDNESDAY)
+            return new PlayTennis();
```

Comment:
For the school sports schedule, you only need to check by month, not by day, as you are only interested in whether it is a summer or winter sport (e.g. the sports day could change but this class should not be affected).